packages/circuits/scripts/benchmark/baseScript.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Ideas:
+ * - create main circom for test/benchmark
+ */
+
+import { mkdtempSync, writeFileSync } from 'fs'
+import * as fastFile from 'fastfile'
+import path from 'path'
+import * as circom from 'circom'
+import * as snarkjs from 'snarkjs'
+
+export async function executeTimeOf(
+    name: string,
+    circomComponent: string,
+    genInputFn: any,
+    genInputArg: any[]
+): Promise<bigint> {
+    // create tmp folder to store circom compile files.
+    const tmpPath = mkdtempSync('/tmp/unirep')
+
+    const circomPath = path.join(
+        __dirname,
+        '../../zksnarkBuild/benchmark.circom'
+    )
+    const r1cs = path.join(tmpPath, `${name}.r1cs`)
```

Comment:
Everytime we create new tmp folder, this mean we don't need create file with`name`.